pe/pe.py

```diff
@@ -1386,6 +1432,11 @@ def fetch_last_content(resource):
                 sub_res.set_heuristic(13)
             res.add_subsection(sub_res)
 
+        if self.binary.resources_manager.langs_available:
```

Comment:
I think you want to negate this?
And you could also use `self.features["resources_manager"]["langs_available"]` instead.
Could you put it toward the top of add_resources where everything related to langs_available is?

Reply:
https://github.com/CybercentreCanada/assemblyline-service-pe/pull/54/commits/6dbc1e182aa7dd013ae1df496b828cc2bf771810